dali/dfuXRefLib/dfuxreflib.cpp

```diff
@@ -2618,18 +2638,29 @@ class CXRefManager: public CXRefManagerBase
             else
                 g.setown(g->combine(gsub.get()));
         }
-        totalSizeOrphans =0;
+        totalSizeOrphans = 0;
         totalNumOrphans = 0;
 
-
         logicalnamelist.kill();
         dirlist.kill();
         orphanlist.kill();
-        
+
+#ifdef _CONTAINERIZED
+        const char *storageDir[1];
+        for (int i = 0; i < nclusters; i++)
+        {
+            const char *storagePlane = clusters[i]; // clusters holds a list of storage plane names
+            storageDir[0] = dirbaselist[i]; // dirbaselist holds the storage plane directories
+
+            loadFromDFS(*this,g,1,storageDir,storagePlane);
+            xrefRemoteDirectories(g,numdirs,storageDir,numthreads);
+        }
```

Comment:
I think this ok, processing 1 {plane + dir} at a time, vs BM, which is processing all at once.
However, 'g' is a melded group of all, so **if** there were non-local storage planes (hostGroup storage) then 'g' would be a combo of all, but each call here is dealing with 1.
So the 'g' combine code above should be moved to BM only, and this should use queryNamedGroupStore().lookup(clusters[i]); directly